Formula/interactive-rebase-tool.rb

```diff
@@ -0,0 +1,15 @@
+class InteractiveRebaseTool < Formula
+  desc "Ncurses sequence editor for git interactive rebase"
+  homepage "https://gitrebasetool.mitmaro.ca/"
+  url "https://github.com/MitMaro/git-interactive-rebase-tool/archive/0.7.0.tar.gz"
+  sha256 "08e5d6dd9beacf7806abd74edfa9e7654ccb2ffc083b2fd8617d132951eee5bd"
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix, "--path", "."
+  end
+
+  test do
+    system "#{bin}/interactive-rebase-tool", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
Did my best here. The app itself is interactive, so I do a simple check by sending the keystroke it expects on stdin.